src/main/java/jenkins/plugins/logstash/persistence/AbstractLogstashIndexerDao.java

```diff
@@ -56,6 +58,17 @@ public AbstractLogstashIndexerDao(String host, int port, String key, String user
     }
   }
 
+  @Override
+  public void setCharset(Charset charset)
```

Comment:
Can you make it explicit what the purpose here is? We have the charset using during the build, and the charset used to encode the outgoing messages. 

Those two do not necessarily need to be the same.

Reply:
Added javadoc to describe the purpose